demisto_sdk/commands/content_graph/parsers/layout.py

```diff
@@ -0,0 +1,64 @@
+from pathlib import Path
+from typing import List
+
+from demisto_sdk.commands.common.constants import MarketplaceVersions
+from demisto_sdk.commands.content_graph.common import ContentType
+from demisto_sdk.commands.content_graph.parsers.content_item import NotAContentItem
+from demisto_sdk.commands.content_graph.parsers.json_content_item import JSONContentItemParser
+
+
+class LayoutParser(JSONContentItemParser, content_type=ContentType.LAYOUT):
+    def __init__(self, path: Path, pack_marketplaces: List[MarketplaceVersions]) -> None:
+        if 'layoutscontainer' not in path.name:
+            raise NotAContentItem
+
+        super().__init__(path, pack_marketplaces)
+        self.kind: str = self.json_data.get('kind')
+        self.tabs: List[str] = self.json_data.get('tabs')
+        self.definition_id: str = self.json_data.get('definitionId')
+        self.group: str = self.json_data.get('group')
+
+        self.edit: bool = bool(self.json_data.get('edit'))
+        self.indicators_details: bool = bool(self.json_data.get('indicatorsDetails'))
+        self.indicators_quick_view: bool = bool(self.json_data.get('indicatorsQuickView'))
+        self.quick_view: bool = bool(self.json_data.get('quickView'))
+        self.close: bool = bool(self.json_data.get('close'))
+        self.details: bool = bool(self.json_data.get('details'))
+        self.details_v2: bool = bool(self.json_data.get('detailsV2'))
+        self.mobile: bool = bool(self.json_data.get('mobile'))
+
+        self.connect_to_dependencies()
+
+    def connect_to_dependencies(self) -> None:
+        """ Collects the incident/indicator fields used as optional dependencies.
+        """
+        if self.group == 'incident':
+            dependency_field_type = ContentType.INCIDENT_FIELD
+        else:
```

Comment:
```suggestion
        elif self.group == 'indicator':
```
And add an `else` block at the bottom, with a log message that an unknown group was found.